### PR TITLE
Changes in Bootzooka GH actions

### PR DESCRIPTION
**Pull request by Pask423**
Split Bootzooka CI action into two separate actions as for now deploy job was always skipped and mergify mark whole action as unsuccessful.

### CODE REVIEW

File: .github/workflows/bootzooka-deploy.yml [hunk 1]
```diff
@@ -15,37 +14,8 @@ on:
...
-          ~/.ivy2/cache
-          ~/.coursier
-        key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.sbt') }}
-
-    - name: Run tests
-      id: run-tests
-      run: sbt test
-
   deploy:
-    if: github.event_name != 'pull_request' && startsWith(github.ref, 'refs/tags/v')
```
Comment (adamw) [inline .github/workflows/bootzooka-deploy.yml:48]: I guess we don't want to run deploy on PRs?

Comment (Pask423) [reply .github/workflows/bootzooka-deploy.yml:48]: I removed PR trigger above in the file 
image

Comment (Pask423) [reply .github/workflows/bootzooka-deploy.yml:48]: startsWith condition should be ok as for now

Comment (adamw) [reply .github/workflows/bootzooka-deploy.yml:48]: ah ok makes sense :)

**Pull request merged**